src/sophys/common/plans/parallel_wrapper.py

```diff
@@ -30,9 +32,10 @@ def parallel_plans_wrapper(*args):
                     if current_thread[list_id] == None:
                         # Yield one message or yield it in a thread in case of a 'wait' message
                         current_event[list_id] = next(parallel_plan_list[list_id])
-                        if current_event[list_id][0] == 'wait':
+                        if current_event[list_id][0] == 'wait' or current_event[list_id][0] == 'sleep':
```

Comment:
As I understood this, we spin up separate threads for only these commands because they are the commands that can block for a while, and we need to execute them at the same time to avoid blocking unnecessarily. You might want to include `wait_for` in this conditional too? And maybe use `current_event[list_id][0] in ['wait', 'sleep', 'wait_for']`?

Reply:
The wait_for is used inside the wait message, and although there is a plan_stub that calls it, I don't think it's used here, but I'll add it in case someone uses it in the future.